src/main/generated/options/soot/options/Options.java

```diff
@@ -100,12 +100,6 @@ public static Options v() {
     public static final int java_version_8 = 9;
     public static final int java_version_1_9 = 10;
     public static final int java_version_9 = 10;
-    public static final int java_version_1_10 = 11;
```

Comment:
Same here. Should be added??

Reply:
This is also not present in develop https://github.com/soot-oss/soot/blob/e233653f2a1bd823d20b490ff75ccc50033ff421/src/main/generated/options/soot/options/Options.java#L102